packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeErrorRenderer.tsx

```diff
@@ -21,60 +21,71 @@ type TreeErrorRendererProps = TreeErrorRendererOwnProps &
   Partial<Pick<ReturnType<typeof useTree>, "getHierarchyLevelDetails">>;
 
 /** @internal */
-export function TreeErrorRenderer({ node, getHierarchyLevelDetails, onFilterClick, reloadTree }: TreeErrorRendererProps) {
-  const { localizedStrings } = useLocalizationContext();
-  if (node.type === "ResultSetTooLarge") {
-    return (
-      <ResultSetTooLargeNode
-        limit={node.resultSetSizeLimit}
-        onOverrideLimit={getHierarchyLevelDetails ? (limit) => getHierarchyLevelDetails(node.parentNodeId)?.setSizeLimit(limit) : undefined}
-        onFilterClick={
-          onFilterClick
-            ? () => {
-                const hierarchyLevelDetails = getHierarchyLevelDetails?.(node.parentNodeId);
-                hierarchyLevelDetails && onFilterClick(hierarchyLevelDetails);
-              }
-            : undefined
-        }
-        aria-level={node.level}
-        aria-posinset={node.posInLevel}
-        aria-setsize={node.levelSize}
-      />
-    );
-  }
-
-  if (node.type === "NoFilterMatches") {
+export const TreeErrorRenderer: React.ForwardRefExoticComponent<TreeErrorRendererProps & RefAttributes<HTMLDivElement>> = forwardRef(
+  ({ node, getHierarchyLevelDetails, onFilterClick, reloadTree }, forwardedRef) => {
```

Comment:
Other props should be passed through to the `Tree.Item` component rendered by this component.

Reply:
changed